components/root/root.jsx

```diff
@@ -280,6 +283,39 @@ export default class Root extends React.PureComponent {
         }
     }
 
+    initOnboardingTaskListForAllUsers = async () => {
+        const currentUser = getCurrentUser(store.getState());
+        const {preferences} = this.props;
+
+        const hasUserStartedOnboarding = preferences.some((pref) =>
+            pref.name === OnboardingTaskList.ONBOARDING_TASK_LIST_INIT && pref.value === 'true');
+
+        if (!hasUserStartedOnboarding) {
```

Comment:
I think we'll also need to check whether existing users already have any progress in the tour, so that we don't re-open things on them if they've started or finished the tour.

Reply:
oh yes, I am checking the former value we used for the next steps too. I just wanted to keep pushing so I can continue validating using the spinwick server.